website/docs/docs/build/udfs.md

```diff
@@ -68,12 +68,13 @@ To define UDFs in dbt, refer to the following steps:
         config:
           schema: udf_schema
           database: udf_db
+          volatility: deterministic
         arguments: # optional
           - name: a_string # required if arguments is specified
             data_type: string # required if arguments is specified
             description: The string that I want to check if it's representing a positive integer (like "10") 
         returns: # required
-          data_type: boolean # required
+          data_type: integer # required
     ```
     </File>
 
```

Comment:
@luna-bianca the following seciton is commented out, but we might want to help Future Us by fixign the example and putting `type` under `config`:
>  functions:
	  - name: string 
	    description: string
	    type: scalar # default value